src/jsr-schemas.ts

```
@@ -0,0 +1,60 @@
+import { z } from 'zod/v4-mini';
+
+// Generated Zod schemas from JSR JSON Schema
+export const JSRScopedNameSchema = z.string().check(
+	z.regex(/^@[a-z0-9\-_]+\/[a-z0-9\-_]+$/),
+);
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)